USF-2530: Update Wishlist documentation after latest updates (#402)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/containers/wishlist-item.mdx

The `WishlistItem` container manages and displays a single product from a wishlist. It allows users to interact with the product, such as moving it to the cart or navigating to the product details page.

> For complex products that have no variant selected in the wishlist, the "Move To Cart" button becomes a "Customize" button, where the system prompts users to select a variant first by navigating to the product details page.

Reviewer: ```suggestion
For complex products that have no variant selected in the wishlist, the **Move To Cart** button becomes a **Customize** button. Users must select a variant first by navigating to the product details page.
```
